packages/protocol/docs/r9-expert-d-output.md

```
@@ -0,0 +1,163 @@
+# Expert D — Coverage + ratchet + test taxonomy
+
+## 1. Coverage gate
+
+- **Provider**: use Vitest `v8`. This is a Node-only TS package, so native V8
+  coverage avoids Istanbul pre-instrumentation. First add the missing provider:
+  ```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)